lxd/storage/utils.go

```diff
@@ -360,3 +384,253 @@ func GetStorageResource(path string) (*api.ResourcesStoragePool, error) {
 
 	return &res, nil
 }
+
+// VolumeTypeNameToType converts a volume type string to internal code.
+func VolumeTypeNameToType(volumeTypeName string) (int, error) {
```

Comment:
This is going to fail if passed any of the snapshot types. Should it return a bool to indicate whether it's a snapshot or not?

Reply:
Ignore me :)